gatorgrade/main.py

```diff
@@ -0,0 +1,19 @@
+import typer
+"""Use Typer to run gatorgrade to run the checks and generate the yml file."""
+
+app = typer.Typer(add_completion=False)
+FILE = "gatorgrade.yml"
```

Comment:
Just leaving this as a note here, we should probably avoid globals. I think a better way to go about it would be to create a config.yml file that has all your default values you need and then use a singleton config module that you can then reference default/configuration properties from. Just a note for the future, maybe make this an issue/discussion if you all agree?

Thoughts @noorbuchi ?

Reply:
I think this is fine for now, we're going with the assumption that the file will always be called `gatorgrade.yml` and that's fine.

Reply:
@ullrichd21 @noorbuchi I'd like to add here that I think GatorGrade should shy away from any configuration that is not in `gatorgrade.yml` or (if absolutely needed) `pyproject.toml`, so that it can work well with the rest of the python ecosystem without adding additional complexity in project structures.